### ISSUE TITLE
Widget marketplace is obsolete

### ISSUE THREAD

**Issue by flopezag** [Wirecloud/wirecloud#323]
The version of different core widgets and operators that the user can obtain from the marketplace are very old and in some cases not working. The user have to go directly to the repository of the corresponding components to download and upload the last version of the components.

**Comment by mjimenezganan** [comment]
Which marketplace do you refer to? I'm wondering if it is the resources section of the WireCloud instance, whose update depends on the specific instance (are you referring mashup.lab.fiware.org?), or the WStore, whose contents are testing only and, though easily downloadable to WireCloud, it is not defined as an stable marketplace but as a playground.

**Comment by brylie** [comment]
Well, I believe I was referring to mashup.lab.fiware.org, but in general a plugin ecosystem with graphical UI would be really useful. There could be one or more repositories, some of which could be enabled by default or at least easy to configure. Some examples include:

- WordPress plugins
- Godot [Asset library](https://godotengine.org/asset-library/asset)

**Comment by mjimenezganan** [comment]
I'll take a note. Having such an official components repository (or customizable repositories sources) is interesting. Lets evaluate the work needed for (1) having such feature inside WC and (2) populating with revised versions of currently dispersed contents.

**Comment by flopezag** [comment]
What I mention and maybe it would be a new functionality is that going to My Resources in the wirecloud, I obtain a list of available widgets and probably they are obsoletes because there are new versions of them. What it should be nice is if you could provide a functionality (a button) in order to check it there are newer available versions of them.

Additionally to it, the documentation of the widget should add the description about from where a user can obtain a new version to be installed.

**Comment by brylie** [comment]
Yeah, another instance is QGIS where you can browse for plugins that are compatible with your version and install/upgrade plugins via the UI.
